code/modules/vore/eating/belly_vr.dm

```diff
@@ -12,7 +12,11 @@
 	var/name								// Name of this location
 	var/inside_flavor						// Flavor text description of inside sight/sound/smells/feels.
 	var/vore_sound = 'sound/vore/gulp.ogg'	// Sound when ingesting someone
+	var/digestion_sounds_pref				// Digestion sound set.
```

Comment:
Datatype of `digestion_sounds_pref` is a list().  since it was named _pref I at first thought it was going to hold a key like "organic" but its a list()  `digestion_sounds_list` maybe would be clearer.  But not a big deal, could also just clarify in the comment.